src/Metadata/Extractor/XmlResourceExtractor.php

```diff
@@ -61,7 +61,6 @@ protected function extractPath(string $path): void
         foreach ($xml->resource as $resource) {
             $base = $this->buildExtendedBase($resource);
             $this->resources[$this->resolve((string) $resource['class'])][] = array_merge($base, [
-                'class' => $this->phpize($resource, 'class', 'string'),
```

Comment:
would it be possible to resolve instead in both extractors? A resource without a class is definitely not valid to me.

Reply:
I wondered about this, but in ApiResource the class field does not exist at all if it is not specified here, as far as I have seen, and at the same time the relevant class is indicated as the key, so setting the class here is a kind of unnecessary duplication